htdocs/admin/stock.php

```diff
@@ -52,14 +52,17 @@
 	// Mode of stock decrease
 	if ($action == 'STOCK_CALCULATE_ON_BILL'
 	|| $action == 'STOCK_CALCULATE_ON_VALIDATE_ORDER'
-	|| $action == 'STOCK_CALCULATE_ON_SHIPMENT')
+	|| $action == 'STOCK_CALCULATE_ON_SHIPMENT'
+	|| $action == 'STOCK_CALCULATE_ON_SHIPMENT_CLASSIFY_BILLED')
```

Comment:
Why calling the option "STOCK_CALCULATE_ON_SHIPMENT_CLASSIFY_BILLED"
The status after "Validated" in shipment is "Closed" (that can mean "shipped or received" depending on user).

Can you call instead option 
STOCK_CALCULATE_ON_SHIPMENT_CLOSE